webrtc.html

```diff
@@ -3059,10 +3078,29 @@ <h2>Methods</h2>
                 apply the supplied
                 <code><a>RTCSessionDescriptionInit</a></code> as the remote
                 offer or answer. This API changes the local media state.</p>
-                <p>When the method is invoked, the user agent MUST return the
-                result of <a href="#set-description">setting the
-                RTCSessionDescription</a> indicated by the method's first
-                argument.</p>
+                <p>When the method is invoked, the user agent MUST run the following steps:</p>
+                <ol>
+                  <li>
+                    <p>Let <var>description</var> be the method's first argument.</p>
+                  </li>
+                  <li>
+                    <p>Let <var>connection</var> be the
+                    <code><a>RTCPeerConnection</a></code> object on which the
+                    method was invoked.</p>
+                  </li>
+                  <li>
+                    <p>Return the result of <a href=
+                    "#enqueue-an-operation">enqueuing</a> the following steps
+                    to <var>connection</var>'s operation queue:</p>
```

Comment:
Same as above: Why are we enqueuing a task to enqueue a task? We should just say to return the result of setting the RTCSessionDescription immediately.